src/main/java/hudson/plugins/analysis/collector/AnalysisGraphConfiguration.java

```diff
@@ -77,7 +77,7 @@ public boolean getCanDeacticateOtherTrendGraphs() {
 
     @Override
     public String serializeToString() {
-        return super.serializeToString() + SEPARATOR + serializeBoolean(canDeactivateGraphs);
+        return super.serializeToString() + serializeBoolean(canDeactivateGraphs);
```

Comment:
Well the upgrade of the dependency to a `SNAPSHOT` is of course related to this PR, but you said the test failures came from the latest _release_, so this could have been fixed in a standalone PR, which could be merged ahead of anything else.

Just a suggestion based on experience with PRs that drag out for a long time. :-)